app/code/Magento/WishlistGraphQl/etc/schema.graphqls

```diff
@@ -19,4 +19,16 @@ type WishlistItem {
     description: String @doc(description: "The customer's comment about this item"),
     added_at: String @doc(description: "The time when the customer added the item to the wish list"),
     product: ProductInterface @resolver(class: "\\Magento\\WishlistGraphQl\\Model\\Resolver\\ProductResolver")
+}
+
+type Mutation {
+    addItemsToWishlist (input: AddItemToWishlistInput): AddItemToWishlistOutput @resolver(class: "\\Magento\\WishlistGraphQl\\Model\\Resolver\\AddItemToWishlist") @doc(description:"Add items by SKU to the wish list")
+}
+
+input AddItemToWishlistInput {
+    skus: [String!]!
+}
+
+type AddItemToWishlistOutput {
+    wishlist: WishlistOutput!
```

Comment:
WishlistOutput => `Wishlist`

Reply:
But we have WishlistOutput everywhere. Should I change that for Query too?